src/dune_rules/findlib.ml

```diff
@@ -484,15 +494,13 @@ type t = DB.t
 
 let create =
   Per_context.create_by_name ~name:"findlib" (fun context ->
-    Memo.lazy_ (fun () ->
```

Comment:
Why did you remove the lazy?

Reply:
Just a small cleanup: It was created and then forced immediately, isn't it a noop?

Reply:
I need to double check, but I don't think so. We're creating a map keyed by the context where every value is the database for that context. We don't need to generate the database for a context until it's actually requested. We are not creating a memo node and immediately forcing it, because the forcing bit is delayed until we look up the context in this map (or at least this is  how it should work).